ConsumerEdition/DragonBoard-410c/Guides/multi-usb-camera.md

```diff
@@ -0,0 +1,53 @@
+---
+title: Using Multiple USB Cameras on a single USB Hub
+permalink: /documentation/ConsumerEdition/DragonBoard-410c/Guides/multi-usb-camera.md.html
+---
+
+# Using Multiple USB Cameras on a single USB Hub
+
+This page provides information to connect and use multiple USB webcams on a single USB port using a USB Hub. This guide assumes that you have installed the latest release of Debian for the Dragonboard 410c.
+
+# Table of Contents
+
+- [1) Software Required](#1-software-required)
+- [2) Hardware Required](#2-hardware-required)
+- [3) Using Multiple USB Cameras](#3-using-multiple-usb-cameras)
+- [4) Footnotes](#4-footnotes)
+
+# 1) Software Required
```

Comment:
Please include commands for update and upgrade. Sometimes, users will just follow these guides out of the box!